### PR TITLE
S02 objectives

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

File: frontend/src/components/LevelPage/GameMenu/GameMenu.jsx [hunk 1]
```diff
@@ -17,11 +17,11 @@ function GameMenu() {
         <img className="JurassiqueCharacter" src={JurassicCharacter} alt="" />
         <hr />
         <button type="button" className="GameMenuButton">
```
Comment (ItsMatDev) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.jsx]: Button into a button

File: frontend/src/components/LevelPage/GameMenu/GameMenu.jsx [hunk 2]
```diff
@@ -17,11 +17,11 @@ function GameMenu() {
...
-          <img className="Bag" src={Bag} alt="Inventaire" />
+        <Inventory />
         </button>
         <hr />
         <button type="button" className="GameMenuButton">
```
Comment (ItsMatDev) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.jsx]: Button into a button

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 3]
```diff
@@ -10,22 +10,34 @@ function Inventory() {
 
   return (
     <>
-      <button type="button" onClick={handleInventoryToggle}>
-        Inventory
-      </button>
+      <div>
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: Div isn't necessary

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 4]
```diff
@@ -10,22 +10,34 @@ function Inventory() {
...
+        <button type="button" onClick={handleInventoryToggle}>
+          <img
+            className="bag"
+            src="./src/assets/pictures/Bag.png"
+            alt="The Bag"
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: In French here

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 5]
```diff
@@ -10,22 +10,34 @@ function Inventory() {
...
-          type="button"
-          className="inventoryPopUp__closeButton"
-          onClick={handleInventoryToggle}
-        >
-          Close
-        </button>
+        <div className="openBag">
+          <img
+            src="./src/assets/pictures/OpenBag.png"
+            alt="Open Bag"
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx:28]: In French here

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 6]
```diff
@@ -10,22 +10,34 @@ function Inventory() {
...
+            id="openBag"
+          />
+          <button
+            type="button"
+            className="inventoryPopUp__closeButton"
+            onClick={handleInventoryToggle}
+          >
+            Close
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: In French here

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 7]
```diff
@@ -18,8 +20,33 @@
...
 }
+
+.openBag {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+}
+
+#openBag {
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: Id is really necessary ?

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+function Objectives() {
+  const [showObjectives, setObjectives] = useState(false);
+
+  const handleObjectivesToggle = () => {
+    setObjectives(!showObjectives);
+  };
+
+  return (
+    <>
+      <div>
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx]: div isn't necessary

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+            alt="List of objectives"
+          />
+        </button>
+      </div>
+      <article
+        className={`objectivesPopUp ${
+          showObjectives ? "objectivesPopUp__show" : ""
+        }`}
+      >
+        <div className="openNotebook">
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx]: Another tag is better

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.openNotebook {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+}
+
+#openChooseLevelPage {
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss]: Id is necessary ?

**Review by Yoann-Fortin — APPROVED** [review]

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 3 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: div deleted

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 5 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx:28]: Amended

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 4 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: Amended

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 6 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: Amended

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss (hunk 7 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: Id changed to className

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx (hunk 8 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx]: Div deleted

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx (hunk 9 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx]: Div changed to section

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss (hunk 10 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss]: Id changed to className

**Pull request merged**